src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java

```diff
@@ -0,0 +1,17 @@
+package org.wso2.maven.humantask.artifact;
+
+/**
+ * Created by heshitha on 9/5/16.
```

Comment:
ASF2.0 license, @author tags will no longer be allowed in any Apache source code. So please remove it.